src/index.html

```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="utf-8">
   <title>EvtViewerAngular</title>
-  <base href="/">
+  <base href="./">
```

Comment:
No don't change this row, restore into <base href="/">
If you need compile with different base path you can use for example ng build --base-href pippo/pluto

Reply:
One thing doesn't exclude the other: without this fix you need to always express the parameter, also live server requires it